### DISCUSSION TITLE
How do I reset a fan to default behavior?

### DISCUSSION THREAD

**Discussion by knuku** [Rem0o/FanControl.Releases#1267] [General]
_Not sure if it goes under 'Q&A' or under 'Ideas', so posting it to General_

First off, awesome app! The only thing that was kind of confusing to me is how to reset a particular fan to its default speed/behavior.

It's not really clear what happens when you don't set any curve to a fan. Apparently, this doesn't shut the fan off—at least some fans with that settings have >0 rpm for my setup. So does it become sensor-based auto mode that way? 

In the fan menu (the three dots one) there is an option to detect the sensor. I selected that for all fans that don't use custom curve, so it looks like I set auto/default behavior for those (which is apparently not reflected in the UI), but I am not really sure if that's the case, could anyone kindly elaborate?

**Comment by Rem0o** [comment]
If you don't enable the toggle switch on a control card, it will be running at default behavior.

**Reply by knuku** [reply]
Gotcha, thanks a bunch! Keep up the good work!
